_articles/fr/2020-08-26-a-la-decouverte-de-mercure.md

```diff
@@ -165,7 +165,7 @@ Bien évidemment, ce protocole est beaucoup plus complet qu'il n'y parait, et il
 
 ![GIF Keep your secrets]({BASE_URL}/imgs/articles/2020-08-26-a-la-decouverte-de-mercure/keep_your_secrets.gif)
 
-Cependant, un codelabs est prévu très bientôt pour approfondir tout cela, et découvrir plein d'autres choses, notamment comment sécuriser les connexions, utiliser Mercure dans une application Symfony... Le tout sur la plateforme [Eleven's Codelabs](https://codelabs.eleven-labs.com/).
+Cependant, un codelabs est prévu très bientôt pour approfondir tout cela, et découvrir plein d'autres choses, notamment comment sécuriser les connexions, utiliser Mercure dans une application Symfony... Le tout sur la plateforme [Eleven's Codelabs](/fr/creez-un-chat-avec-symfony-et-mercure/).
```

Comment:
i see some links for codelabs with `{BASE_URL}` like https://github.com/eleven-labs/blog.eleven-labs.com/pull/1045/files#diff-626ff138ff10ddebefea919de55180f16c1ed6986a32b2f2bed69a3828e99d4eR29 and some without which one is good ?

Reply:
it works fine http://dev.blog.eleven-labs.com.s3-website.eu-west-2.amazonaws.com/fix/broken-links/fr/creez-un-chat-avec-symfony-et-mercure/

Reply:
Both works but better use always BASE_URL in order for all links to work in all envs including dev. I'm gonna fix the ones I've missed